### DISCUSSION TITLE
How many ducks are there?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#396] [Galaxy Trivia] [answered]
There are two ducks in front of a duck, two ducks behind a duck and a duck in the middle.

**Comment by peterbaccaro** [comment]
**Three** ducks.  They are all in a row, so ducks 1 and 2 are in front of duck 3, ducks 2 and 3 are behind duck 1 and duck 2 is in the middle.

**Comment by Roberto-Larivera** [comment]
3 Anatre

**Comment by thuongtruong1009** [comment, marked as answer]
There are **3 ducks** in total. The first two ducks are in front of the third duck, and the last two ducks are behind the third duck. The description of "a duck in the middle" refers to the same third duck that is both preceded and followed by two other ducks.
